src/mds/journal.cc

```diff
@@ -1620,8 +1620,9 @@ void EMetaBlob::replay(MDSRank *mds, LogSegment *logseg, int type, MDPeerUpdate
 	if (used_preallocated_ino) {
 	  if (!session->info.prealloc_inos.empty()) {
 	    inodeno_t ino = session->take_ino(used_preallocated_ino);
-	    session->info.prealloc_inos.erase(ino);
+            dout(10) "received ino " << ino " from the session" << dendl;
```

Comment:
If we are printing this for crash debug, then let's do dout(5)?

Reply:
done